src/onepasswordconnectsdk/client.py

```diff
@@ -1,55 +1,34 @@
 """Python Client for connecting to 1Password Connect"""
-from dateutil.parser import parse
+import httpx
+from httpx import HTTPError
 import json
 import os
-import re
-import requests
-import datetime
-from requests.exceptions import HTTPError
-import onepasswordconnectsdk
+
+from onepasswordconnectsdk.serializer import Serializer
+from onepasswordconnectsdk.utils import is_valid_uuid
+from onepasswordconnectsdk.errors import (
+    FailedToRetrieveItemException,
+    FailedToRetrieveVaultException,
+    EnvironmentHostNotSetException,
+    EnvironmentTokenNotSetException,
+)
 from onepasswordconnectsdk.models import Item, ItemVault
 from onepasswordconnectsdk.models.constants import CONNECT_HOST_ENV_VARIABLE
 
 ENV_SERVICE_ACCOUNT_JWT_VARIABLE = "OP_CONNECT_TOKEN"
-UUIDLength = 26
+ENV_IS_ASYNC_CLIENT = "OP_CONNECT_CLIENT_ASYNC"
```

Comment:
I haven't seen in the code, so I may not know everything, but how do we make sure that this value defaults to false to keep the backwards-compatibility?

Reply:
Awesome, thank you for highlighting this. 😄